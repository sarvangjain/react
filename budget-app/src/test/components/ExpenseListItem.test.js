import ExpenseListItem from '../../components/ExpenseListItem';
import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses';
import toJSON, { toJson } from 'enzyme-to-json';

test('should render Expense list item', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})