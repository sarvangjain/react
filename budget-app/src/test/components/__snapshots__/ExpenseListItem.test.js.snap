// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense list item 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    $1.95
    -
    1st, Jan '70
  </p>
</div>
`;
